### PR TITLE
Migrate PHPUnit configuration

### PR DESCRIPTION
**Pull request by DannyvdSluijs**
This PR migrates the PHPUnit configuration as suggested by PHPUnit when running it.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-        <testsuite name="Unit Tests">
-            <directory>tests</directory>
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist>
-            <directory suffix=".php">src/</directory>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/autoload.php" backupGlobals="false" backupStaticAttributes="false" colors="true" verbose="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```
Comment (mfn) [inline phpunit.xml.dist:2]: Can we keep this split over multiple lines for readability as not to have to scroll 400 characters to the right to read everything?

**Review by mfn — APPROVED** [review]

**Comment by DannyvdSluijs** [comment]
Yeah sure will update the PR tomorrow.

**Comment by DannyvdSluijs** [comment]
I've updated the PR and reworked the update configuration a bit to reduce the diff compared to the old version.

**Review by mfn — APPROVED** [review]
Thank you, that makes much more sense!

ping @barryvdh

**Pull request merged**